source/components/molecules/RepeaterField/RepeaterFieldListItem.tsx

```diff
@@ -191,6 +199,8 @@ const RepeaterFieldListItem: React.FC<Props> = ({
             <InputComponent 
               input={input} 
               onChange={changeFromInput(input)} 
+              error={error && error[input.id] ? {isValid: error[input.id].isValid, message: error[input.id].validationMessage} : undefined}
+              showErrorMessage={error && error[input.id] && !error[input.id].isValid}
```

Comment:
This is a bit wordy for an inline-expression. Could we extract this to two variables, one for the error object and a flag for if we want to show an error respectively?